src/mmw/js/src/core/templates/modificationPopup.html

```diff
@@ -11,6 +11,6 @@
 </table>
 <span>
     <button class="btn btn-sm btn-icon dark delete-modification" type="button">
-        <i class="fa fa-trash"></i>
+        <i class="fa fa-trash"></i> Remove modification
```

Comment:
Consider adding the same [here](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/templates/gwlfeScenarioToolbar.html#L43)

![screen shot 2017-05-25 at 5 05 31 pm](https://cloud.githubusercontent.com/assets/7633670/26498437/0dbbc09e-41fd-11e7-9a53-f1d47345f21d.png)